CHANGELOG.md

```diff
@@ -1,18 +1,32 @@
+# 4.0.0 (2020-05-07)
+
+### New features
+
+- Updated the pipeline to transpile to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview). [#11](https://github.com/blackbaud/skyux-assets/pull/11)
```

Comment:
Why do have this item again after we already had it in February?

Also, is #11 the correct PR? When I open it up, I don't see anything about the Angular Packet Format. And it looks like the changes are just to RC branches while this changelog entry sounds like we're announcing the changes outside of RC and in the master branch.

Reply:
For major version releases it's good practice to combine all changes from previous pre-releases so that the consumer knows what enhancements, bug fixes, and breaking changes are included in the next major version.

It's the correct PR, yes. The new package format was introduced in a specific version of Builder (in this case, a specific branch of Builder). After that PR was released, Builder used `ng-packagr` internally to build the library in a different format than it had been built before.